README.rst

```diff
@@ -173,18 +187,9 @@ SoCo supports the following controls amongst others:
    -  MAC Address
 
 -  Set the speaker’s Zone Name
--  Find all the Sonos speakers in a network.
```

Comment:
Did you miss this one when you re-factored, or is it on purpose, because of all the examples?

Reply:
I left it out since all the other actions only work if you already have the SoCo instance, and the ability to discover speakers is already well-presented in the Examples section.

Reply:
All right.